### DISCUSSION TITLE
When will 1.8.0 come out?

### DISCUSSION THREAD

**Discussion by Marc92020** [ScratchAddons/ScratchAddons#1392] [Help] [answered]
It's due by today and it is still not 1.8.0. It's only 1.7.1 still. What happened to it?

**Comment by pufferfish101007** [comment, marked as answer]
The extension's usually uploaded to the extension stores on the sunday or monday. I think the point of the milestone is to not merge anything new until tge extension's in the stores so that translators have time to translate any new strings.

**Reply by Marc92020** [reply]
ok
